src/concrete/ml/sklearn/linear_model.py

```diff
@@ -347,7 +347,7 @@ def _get_training_quantized_module(
         # Enable the underlying FHE circuit to be composed with itself
         # This feature is used in order to be able to iterate in the clear n times without having
         # to encrypt/decrypt the weight/bias values between each loop
-        configuration = Configuration(composable=True, compress_evaluation_keys=True)
```

Comment:
the feature is activated in the QuantizedModule's compile function, so let's avoid some confusion here